src/fides/api/util/saas_util.py

```diff
@@ -349,7 +349,8 @@ def assign_placeholders(value: Any, param_values: Dict[str, Any]) -> Optional[An
             # removes outer {} wrapper from body for greater flexibility in custom body config
             if isinstance(placeholder_value, dict):
                 placeholder_value = json.dumps(placeholder_value)[1:-1]
-
+            if isinstance(placeholder_value, list):
+                placeholder_value = json.dumps(placeholder_value)
```

Comment:
oh wow `assign_placeholders` this is a function I haven't looked at it a long time -